php/utils.php

```diff
@@ -201,6 +201,24 @@ function locate_wp_config() {
 	return $path;
 }
 
+function locate_wp_config_sample() {
```

Comment:
Can we make this method private in `WP_CLI_Core_Command` so we don't expose it as a public utility?